packages/quality-gates/hooks/quality_gate_config.py

```
@@ -59,6 +64,45 @@
     "warned": False,
 }
 
+# .claude/state/ 配下に状態ファイルを解決する既定ディレクトリ。
+# evaluation-set-checker.py と同じ規約（audit-flags.json の paths.state_dir で
+# 上書き可能）。/tmp のグローバル共有ファイルはプロジェクト外に置かれるため、
+# パストラバーサル対策込みで worktree（= project_dir）配下に閉じ込める。
+DEFAULT_STATE_DIR = os.path.join(".claude", "state")
+
+
+def resolve_state_path(project_dir: str, filename: str, config: dict | None = None) -> str:
+    """<project_dir>/.claude/state/<filename> に解決する（paths.state_dir 上書き対応）。
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)